buildbot.mariadb.org/constants.py

```diff
@@ -1,12 +1,12 @@
-github_status_builders = ["amd64-centos-7", "amd64-debian-10", "amd64-fedora-35", "amd64-ubuntu-2004-clang11", "amd64-windows", "amd64-windows-packages"]
+github_status_builders = ["amd64-centos-7", "amd64-debian-10", "amd64-fedora-35", "amd64-fedora-36", "amd64-ubuntu-2004-clang11", "amd64-windows", "amd64-windows-packages"]
```

Comment:
This should not be changed. We only want to report status to GitHub for protected branches builders and Fedora 36 will not be part of it for now.

Reply:
Removed Fedora 36 from Github status report.